style.css

```diff
@@ -330,3 +352,118 @@ textarea.input {
   height: 4px;
   margin-top: 120px;
 }
+
+.desktop-nav {
+  display: none;
+}
+
+@media (min-width: 800px) {
```

Comment:
- [ ] Please replace min-width with proper values, make sure to use any value between 992 and 110, because `800px` is too small 🤔